tests/demo_with_dash/build.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
Don't copy, just make name configurable with an env-var.

Reply:
That will not work. The name contains '-', but pkg_name does not. See logic, plugin, and other variables that use pkg_name instead of name.

Reply:
It will. I've done it for you in https://github.com/tox-dev/tox-uv/pull/33, and also demonstrating I believe that the bug is not in our tool but your build backend violating PEP-427.